adapters/alkimi/alkimi_test.go

```diff
@@ -0,0 +1,181 @@
+package alkimi
+
+import (
+	"encoding/json"
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"testing"
+
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	alkimiTestEndpoint = "https://exchange.alkimi-onboarding.com/server/bid"
+)
+
+func TestEndpointEmpty(t *testing.T) {
+	_, buildErr := Builder(openrtb_ext.BidderAlkimi, config.Adapter{
+		Endpoint: ""}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
Should use actual endpoint mentioned in bidder-info yaml. This is to make sure adapter code is compatible and working correctly with actual endpoint.

Reply:
fixed